src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/MasterServiceMetricsTests.java

```diff
@@ -116,8 +113,8 @@ private String readMetricsInJsonString(int size) {
         assert metrics.size() == size;
         if (size != 0) {
             String[] jsonStrs = metrics.get(0).value.split("\n");
-            assert jsonStrs.length == 2;
-            return jsonStrs[1];
+            assert jsonStrs.length == 1;
```

Comment:
Why we need to change the length from 2 to 1? Also, can we move this method to TestUtil.java?

Reply:
If we don't have any pending tasks then we are not putting any data in tmp file. Earlier, If Pending tasks are 0 then were writing {"Master_PendingQueueSize":0} to the tmp file.

Earlier metric variable looks like
```
[/dev/shm/performanceanalyzer/1153710000/pending_tasks/current/metadata:1153720000::{"current_time":1616703285630}
{"Master_PendingQueueSize":0}]
```
Now metric variable looks like. This is the reason for changing 2 to 1
```
[/dev/shm/performanceanalyzer/1153710000/pending_tasks/current/metadata:1153720000::{"current_time":1616703140203}]
```